src/views/home/sections/Sponsors/Sponsors.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from "next/image";
+
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+import sponsors from "assets/data/sponsors.json";
+import logo_stickerMule from "assets/sponsors/sticker-mule-logo-light-bg-stacked.svg";
+
+import styles from "./Sponsors.module.scss";
+
+interface SponsorItemProps {
+	name: string;
+	src: string;
+	url: string;
+	bg?: string;
+}
+
+function SponsorItem({ name, src, url, bg }: SponsorItemProps) {
+	return (
+		<a href={url}>
+			<Image
+				src={src}
+				alt={name}
+				className={styles.smallTierImage}
+				style={{ backgroundColor: bg }}
+			/>
+		</a>
+	);
+}
+
+function Sponsors() {
+	// Temporary workaround
+	const stickerMule = sponsors.smallTier[4];
+
+	return (
+		<section id="sponsors" className="container">
+			<h2>Sponsors</h2>
+			<p className={styles.sponsorInfo}>
+				Interested in sponsoring HackUCI 2023? Check out our information above
+				to learn more about our event! For more information, please email us
```

Comment:
Suggestion: use of word "information" seems a little redundant/conflicting. Perhaps say "For additional inquiries" in second sentence.